partial.html

```diff
@@ -0,0 +1,121 @@
+---
+id: partial
+title: BCDA Partial Adjudicated Claims
+layout: default
+---
+<div class="ds-base ds-l-container">
+  <div role="region" aria-label="page-title" class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      BCDA Partial Adjudicated Claims
+    </div>
```

Comment:
Same as earlier question, should this be "Partially Adjudicated"?
```suggestion
id: partial
title: BCDA Partially Adjudicated Claims
layout: default
---
<div class="ds-base ds-l-container">
  <div role="region" aria-label="page-title" class="ds-l-row ds-u-fill--primary-darker">
    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
      BCDA Partially Adjudicated Claims
    </div>
```